Tasks/AzureMysqlDeployment/task.json

```diff
@@ -0,0 +1,231 @@
+            "isExpanded": true
+        },
+        {
+            "name": "firewall",
+            "displayName": "Firewall",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "ConnectedServiceName",
+            "aliases": ["azureSubscription"],
+            "type": "connectedService:AzureRM",
+            "label": "Azure Subscription",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "This is needed to connect to your Azure account.<br>To configure new service connection, select the Azure subscription from the list and click 'Authorize'.<br>If your subscription is not listed or if you want to use an existing Service Principal, you can setup an Azure service connection using 'Add' or 'Manage' button."
+        },
+        {
+            "name": "ServerName",
+            "type": "pickList",
+            "label": "Host Name",
+            "required": true,
+            "defaultValue": "",
+            "groupName": "target",
+            "helpMarkDown": "Server name of 'Azure DB for Mysql'. Example: fabrikam.mysql.database.azure.com​.",
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "DatabaseName",
+            "type": "string",
+            "label": "Database Name",
+            "required": false,
+            "groupName": "target",
+            "defaultValue": "",
+            "helpMarkDown": "The name of database on which the script is needed to be run​."
+        },
+        {
+            "name": "SqlUsername",
+            "type": "string",
+            "label": "Server Admin Login",
+            "required": false,
+            "groupName": "target",
+            "defaultValue": "",
+            "helpMarkDown": "Azure Database for MySQL server supports native MySQL authentication.<br>You can connect and authenticate to a server with the server's admin login. Example:  bbo1@fabrikam."
+        },
+        {
+            "name": "SqlPassword",
+            "type": "string",
+            "label": "Password",
+            "required": false,
+            "groupName": "target",
+            "defaultValue": "",
+            "helpMarkDown": "Administrator password for Azure DB for Mysql. In case you don’t recall the password you can change the password from [Azure portal](https://docs.microsoft.com/en-us/azure/mysql/howto-create-manage-server-portal).<br>It can be variable defined in the definition. Example : $(password).<br>Also, you may mark the variable type as 'secret' to secure it."
+        },
+        {
+            "name": "TaskNameSelector",
+            "type": "pickList",
+            "label": "Type",
+            "required": false,
+            "helpMarkDown": "",
+            "groupName": "taskDetails",
+            "defaultValue": "SqlTask",
+            "options": {
+                "SqlTask": "MySQL Script File",
```

Comment:
suggestion to rename  from `SqlTask` to `SqlTaskFile`

Reply:
Done.